.github/ISSUE_TEMPLATE/cast-idea.yml

```diff
@@ -0,0 +1,9 @@
+name: Cast Idea
+description: Write down your idea for a NixCast.
+title: "[Idea]: "
+labels: ["idea"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank your for taking the time to give your idea for a NixCast!
```

Comment:
What's this?

Reply:
Nope, just noticed that as well. I used the `.github` folder from another repo. 😅